Documentation/operations/system_requirements.rst

```diff
@@ -277,6 +277,7 @@ VXLAN Tunnel Endpoint (VTEP) Integration               >= 5.2
 Full support for :ref:`session-affinity`               >= 5.7
 BPF-based proxy redirection                            >= 5.7
 Socket-level LB bypass in pod netns                    >= 5.7
+L3 devices                                             >= 5.8
```

Comment:
@brb Is that the proper way to refer to this feature? Do we have any documentation on this I could link to?

Reply:
Unfortunately noup.